src/background/Wallet/Wallet.ts

```diff
@@ -223,9 +224,13 @@ export class Wallet {
         emitter.emit('globalPreferencesChange', state, prevState);
       })
     );
+    // todo: group into "syncWithExternalStores" or "updateExternalStores"?
     this.disposer.add(
       this.walletStore.on('change', this.updateChainConfigStore.bind(this))
     );
+    this.disposer.add(
+      this.walletStore.on('change', this.updateLastAddress.bind(this))
+    );
```

Comment:
(I would postpone grouping/extracting until we need to have the same calls anywhere else)

Reply:
> until we need to have the same two calls anywhere else

But we do :) On lines 255-256, that's the reason for the suggestion